alpine/Dockerfile

```diff
@@ -19,28 +18,29 @@ ARG LOCAL_KONG_PACKAGE=empty-kong.tar.gz
 COPY $LOCAL_KONG_PACKAGE /tmp/kong.tar.gz
 
 RUN if [ "$ASSET_LOCATION" = "local" ] ; then exit 0 ; fi; \
-    . /tmp/build.sh
+    apk add --no-cache --virtual .build-deps curl wget tar ca-certificates && \
+    . /tmp/build.sh && \
+    apk del .build-deps
 	
 RUN mkdir /kong; \
-	tar -C /kong -xzf /tmp/kong.tar.gz
-
-FROM alpine:3.11
-
-LABEL maintainer="Kong <support@konghq.com>"
-
-COPY --from=downloader /kong/usr/local /usr/local
-COPY --from=downloader /kong/etc /etc
+	tar -C /kong -xzf /tmp/kong.tar.gz && \
+	mv /kong/usr/local/* /usr/local && \
+	mv /kong/etc/* /etc && \
+	rm -rf /kong
 
 RUN apk add --no-cache libgcc openssl pcre perl tzdata libcap zip \
     && adduser -S kong \
     && mkdir -p "/usr/local/kong" \
 	&& chown -R kong:0 /usr/local/kong \
+	&& chown kong:0 /usr/local/bin/kong \
 	&& chmod -R g=u /usr/local/kong
 
-USER kong
-
 COPY docker-entrypoint.sh /docker-entrypoint.sh
 
+RUN chmod 777 /docker-entrypoint.sh
```

Comment:
Is this 777 required?

Reply:
It was on my local but I think it's because something has (mis)configured security setup. I'll nuke it and if it passes CI consider it OK :)